pandas/tests/io/formats/style/test_style.py

```diff
@@ -187,6 +187,12 @@ def test_render_trimming_mi():
     assert {"attributes": 'colspan="2"'}.items() <= ctx["head"][0][2].items()
 
 
+def test_render_empty_mi():
+    # GH 43305
+    df = DataFrame(index=MultiIndex.from_product([["A"], [0, 1]], names=[None, "one"]))
+    df.style.to_html()
```

Comment:
Worth asserting this is as expected?

Reply:
not convinced. but I added it anyway. 👍

Reply:
Thanks - will save any regressions over time.